dev/com.ibm.ws.cdi.third.party_fat/fat/src/com/ibm/ws/cdi12/fat/tests/HibernateSearchTest.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2015, 2018 IBM Corporation and others.
+ * Copyright (c) 2019 IBM Corporation and others.
```

Comment:
You shouldn't change the copyright like this... even if there were any other changes in the file

Reply:
The file was created in 2019, I noticed that when writing this PR and corrected the date.

Reply:
ok